### ISSUE TITLE
Dictionary Editor: Raw Steno Input

### ISSUE THREAD

**Issue by Wilfried84** [openstenoproject/plover#750]
When entering a "Filter by stroke" search in the Plover dictionary editor, you now have to fingerspell the raw steno outline in order to search it. You have be able to stroke in the search box and have output show up as raw steno. This is already the behavior in the "Strokes" box in the Add Translation window, and it would make sense for "Filter by stroke" too. 

It would also be nice if "Lookup" also offered  a reverse lookup "Lookup by stroke" option.

**Comment by gdwaynewarner** [comment]
Yes, that would be a great feature! digitalCAT's Dictionary Maintenance program has something similar,  but I don't know for sure if you can user your steno machine to add the strokes directly (it's been a while).  When you ser your computer keyboard, however, it will only accept certain letters in the appropriate fields, as (kinda-sorta) shown below:

![screen shot 2017-06-23 at 12 29 41 pm](https://user-images.githubusercontent.com/17680111/27497734-f8d1ba38-5810-11e7-84a6-e483847dc03f.png)

I don't know if you can see it in the above screenshot, but the text entry area is separated into two sections, so if you type the letter D on the left half, you get TK, but if you type a D on the right side, you get a D.

Unfortunately, since I am running this program under Wine, I have not tried using my writer to enter the letters to make an addition to this digitalCAT copy of my dictionary -- which, for the curious, is how I back up my dictionary entries for the eventual time when I start using regular CAT software again --so I'm not sure how it responds to input from the steno keyboard.
